pricelist_cache/models/product_pricelist.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from datetime import date
+
+from odoo import api, fields, models
+
+
+class Pricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    parent_pricelist_ids = fields.Many2many(
+        "product.pricelist",
+        relation="product_pricelist_cache__parent_pricelist_ids_rel",
+        column1="pricelist_id",
+        column2="parent_pricelist_id",
+        compute="_compute_parent_pricelist_ids",
+        store=True,
+    )
+    is_pricelist_cache_computed = fields.Boolean()
+    is_pricelist_cache_available = fields.Boolean(
+        compute="_compute_is_pricelist_cache_available", store=True
```

Comment:
```suggestion
        compute="_compute_is_pricelist_cache_available",
        store=True,
        recursive=True,
```
_/opt/odoo/odoo/fields.py:746: UserWarning: Field product.pricelist.is_pricelist_cache_available should be declared with recursive=True_